### PR TITLE
Initial import

### PR DESCRIPTION
**Pull request by sagikazarmark**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
         "docs": "https://portphp.readthedocs.org"
     },
     "require": {
-        "php": ">=5.4.0"
+        "php": ">=5.4.0",
+        "portphp/portphp": "dev-master"
```
Comment (ddeboer) [inline composer.json]: Please require 1.0.x-dev instead.

Comment (sagikazarmark) [inline composer.json]: I used 1.0@dev, which is the composer recommendation.

Comment (ddeboer) [inline composer.json]: Didn’t know that. Where did you find it?

Comment (sagikazarmark) [inline composer.json]: On packagist :stuck_out_tongue_winking_eye: 

`"portphp/portphp": "1.0.*@dev"` (under the master branch)

**Comment by Tas12345** [comment]
special-sniffle

**Pull request merged**